src/initialization/MOM_state_initialization.F90

```diff
@@ -67,7 +67,7 @@ module MOM_state_initialization
 use MOM_regridding, only : regridding_CS
 use MOM_remapping, only : remapping_CS, remapping_core, initialize_remapping
 use MOM_remapping, only : dzFromH1H2, remapDisableBoundaryExtrapolation
-
+use MOM_tracer_initialization_from_Z, only : horiz_interp_and_extrap_tracer, MOM_initialize_tracer_from_Z
```

Comment:
I thought we were going to use the new routine for T and S initialization too - in order to test it?